.github/workflows/tests.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v4
+      - uses: actions/setup-python@v5
         with:
           python-version: "3.11"
```

Comment:
Maybe we should start running these with 3.12 (not related to this PR though).

Reply:
I'll send a separate PR, just to keep this simple and separate.